vtex-sites/nextjs.store
# Feat: Uses Hero component from FSUI

## What's the purpose of this pull request?

This PR is part of the Component Placement project. Moves `Hero` component to FSUI and replace it. 

## How does it work?

- Uses `Hero` component from FSUI and remove local one.
- Moves `hero.scss` to `src/components/sections/Hero/`
- Renames `data-fs-hero-content` token to `data-fs-hero-heading`

## How to test it?

Nothing should change visually on the component.

Check on the [preview](https://nextjs-store-storybook-git-feat-fsss-351-replace-hero-faststore.vercel.app/) if the Hero is working fine as [before](https://nextjs.vtex.app/).

## Checklist

<em>You may erase this after checking them all ;)</em>
- [x] After https://github.com/vtex/faststore/pull/1336 is merged, update faststore/ui version on stores
